### DISCUSSION TITLE
How to import a schema.prisma file inside another?

### DISCUSSION THREAD

**Discussion by arioberek** [prisma/prisma#12326] [Q&A] [answered]
Hello!

One of my schema.prisma's file is wrote like this:

```
generator client {
  provider = "prisma-client-js"
  output   = "./generated/own_database"
}

datasource db {
  provider = "postgresql"
  url      = env("DATABASE_URL")
}

model Employee {
  ...
}
```

And I have another one, which is like this:

```
generator client {
    provider = "prisma-client-js"
    output   = "./generated/another_database"
}

datasource db {
    provider = "postgresql"
    url      = env("DATABASE_URL_2")
}

model Costs {
    cod_center_cost Int      @id
    description String?  @db.VarChar(100)
    status           Boolean?
    classification Int?
}

...

```

I have a model that needs to have a relation to another model, but how can I refer an another file's model?

**Comment by austincrim** [comment, marked as answer]
Hey @arrbrk1 👋 ,

Prisma does not support this pattern right now. We have a number of outstanding feature requests that relation to your question:

1. https://github.com/prisma/prisma/issues/2377
2. https://github.com/prisma/prisma/issues/1122
3. https://github.com/prisma/prisma/issues/2443

I would encourage you to add your +1 to those issues to help our engineering teams prioritize.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
